plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/ThemeConfig.java

```diff
@@ -80,8 +100,17 @@ public void setHeadJs(List<String> headJs) {
      * Sets the footer JS relative paths of this theme configuration.
      *
      * @param js footer JS relative paths to be set
+     * @throws IllegalArgumentException if a JS file relative path starts with a '/'
      */
     public void setJs(List<String> js) {
+        if (js != null) {
+            for (String relativePath : js) {
+                if (relativePath.charAt(0) == '/') {
+                    throw new IllegalArgumentException(
+                            "JS file relative path cannot start with a '/'. '" + relativePath + "' is incorrect.");
```

Comment:
I think this way looks better as the whole message is in a one line, which improves the readability.